storage_service/locations/signals.py

```diff
@@ -35,8 +39,16 @@ def report_deletion_request(sender, **kwargs):
     _notify_administrators(subject, message)
 
 
+def _log_report(uuid, success, message=None):
+    package = models.Package.objects.get(uuid=uuid)
+    models.FixityLog.objects.create(package=package, error_details=message)
```

Comment:
Shouldn't `success` be used here?

This can also be done with `models.FixityLog.objects.create(package_id=uuid, error_details=message)`.  Is it worth having a 1-line cleanup function?